extensions/apps/extensions/src/components/pages/install-extension/install-terms-conditions.tsx

```diff
@@ -198,16 +198,18 @@ export const ExtensionInstallTerms = ({ appId }: { appId: string }) => {
       <Stack direction="column" spacing={4} className="mx-4 mb-4">
         {Object.keys(acceptedTerms).map(stateKey => (
           <Stack direction="row" spacing={2} key={stateKey} alignItems={'center'}>
-            <Checkbox
-              key={stateKey}
-              id={stateKey}
-              isSelected={acceptedTerms[stateKey]}
-              name={stateKey}
-              label={fieldLabels[stateKey]}
-              handleChange={handleCheckboxChange}
-              value={'value'}
-              customStyle={'gap-x-3'}
-            />
+            <Stack direction="row" alignItems="center" spacing={2}>
+              <Checkbox
+                id={stateKey}
+                name={stateKey}
+                value={'value'}
```

Comment:
better to assign 'stateKey' also to value so the mapped items don't share same value?